planetiler-core/src/main/java/com/onthegomap/planetiler/config/PlanetilerConfig.java

```diff
@@ -166,6 +167,7 @@ public static PlanetilerConfig from(Arguments arguments) {
         "Planetiler downloader (https://github.com/onthegomap/planetiler)"),
       arguments.getDuration("http_timeout", "Timeout to use when downloading files over HTTP", "30s"),
       arguments.getInteger("http_retries", "Retries to use when downloading files over HTTP", 1),
+      arguments.getDuration("http_retry_wait", "How long (in milliseconds) to wait before retrying HTTP request", "5s"),
```

Comment:
Looks good! Can you just remove "in milliseconds" now?

Reply:
Yup, sorry. You're quick. Thanks.